WORKSPACE

```diff
@@ -1,3 +1,5 @@
+workspace(name = "org_tensorflow_serving")
```

Comment:
Workspace names are supposed to be in reverse domain name form, so I don't think tf_serving really makes sense.